x-pack/legacy/plugins/siem/public/components/loader/loader.tsx

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
I feel like this component should belong inside of index.ts

Reply:
I've seen this go both ways within the `/siem` product. Is there a particular reason why one is better than the other? I have no problem changing it...just curious :)

Reply:
I feel like when we try to export only one component there is no need of the index just doing the export.

Reply:
👍